webapp/src/components/sidebar_right/index.tsx

```diff
@@ -127,6 +127,7 @@ function SidebarRight({theme}: {theme: Theme}) {
         };
     });
 
+    // States used for storing the updated details of PRs, reviews, and RHSState after being taken from the redux store.
     const [updatedPrs, setUpdatedPrs] = useState<Item[]>(yourPrs);
```

Comment:
I think a better way to name this would be 
```suggestion
    const [prsWithDetails, setPrsWithDetails] = useState<Item[]>(yourPrs);
```